scheduler/src/cook/kubernetes/api.clj

```diff
@@ -1815,26 +1820,27 @@
 (defn pod-unschedulable?
   "Returns true if the given pod status has a PodScheduled
   condition with status False and reason Unschedulable"
-  [pod-name ^V1PodStatus pod-status]
+  [pod-name ^V1PodStatus pod-status ^V1Pod pod]
   (let [{:keys [pod-condition-unschedulable-seconds
                 synthetic-pod-condition-unschedulable-seconds]}
         (config/kubernetes)
         unschedulable-seconds
-        (if (some-> pod-name synthetic-pod?)
+        (if (or (some-> pod-name synthetic-pod?)
```

Comment:
I think we should split this up into two cases:
* `synthetic pod unschedulable seconds`
* `k8s-packed-pod-unschedulable-seconds`

Reply:
I originally had this as a separate configuration. However, once we moved scheduler config to be per pool, it became much more involved to get this setting at this level of the code. I would need to get the pool for the pod and use the pool regex matcher to extract the setting. Would you be ok with this being globally configured on `:kubernetes`?